projboard/views.py

```diff
@@ -51,6 +51,24 @@ def create_article(request):
     })
 
 
+def my_articles(request):
+    # TODO when we end to create authentication and authorization, change "User1"
+    user = User.get_user_by_nickname("User1")
+    my_articles = Article.search_by_user(user)
+    num_articles = len(my_articles)
+
+    if request.method == "GET":
+        return render(request, 'myArticles/my_articles.html', {
+            'my_articles': my_articles,
+            'num_articles': num_articles})
+    else:
+        raise Http404()
+
+    return render(request, 'myArticles/my_articles.html', {
```

Comment:
Is this code needed? 
you have else with 404,
and then you return the articles. can you explain?